packages/shared/src/components/sidebar/sections/BookmarkSection.tsx

```diff
@@ -1,46 +1,85 @@
 import React, { ReactElement } from 'react';
 import { ListIcon, SidebarMenuItem } from '../common';
-import { BookmarkIcon, PlusIcon } from '../../icons';
+import { ArrowIcon, BookmarkIcon, PlusIcon } from '../../icons';
 import { Section } from '../Section';
 import { webappUrl } from '../../../lib/constants';
 import { SidebarSettingsFlags } from '../../../graphql/settings';
 import { SidebarSectionProps } from './common';
 import { useLazyModal } from '../../../hooks/useLazyModal';
 import { LazyModal } from '../../modals/common/types';
+import { BookmarkReminderIcon } from '../../icons/Bookmark/Reminder';
+import {
+  useBookmarkFolderList,
+  useCreateBookmarkFolder,
+} from '../../../hooks/bookmark';
+import { useViewSize, ViewSize } from '../../../hooks';
 
 export const BookmarkSection = ({
   isItemsButton,
   ...defaultRenderSectionProps
 }: SidebarSectionProps): ReactElement => {
   const { openModal } = useLazyModal();
+  const { folders } = useBookmarkFolderList();
+  const { createFolder } = useCreateBookmarkFolder();
+
+  const isLaptop = useViewSize(ViewSize.Laptop);
+  const rightIcon = !isLaptop && (() => <ArrowIcon className="rotate-90" />);
+
+  const onAddFolderClick = (
+    event: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>,
+  ) => {
+    event.preventDefault();
+    openModal({
+      type: LazyModal.BookmarkFolderSoon,
+      props: {
+        onAfterClose() {
+          createFolder({
+            name: 'New folder',
+            icon: '🚀',
+          });
```

Comment:
This is temporary?

Reply:
Yes, just need for now until the creation modal is ready

Reply:
Removed ✅